packages/bridge-ui-v2/src/components/Transactions/Transaction.svelte

```diff
@@ -65,7 +65,7 @@
   {/if}
 
   <div class="sm:w-1/4 md:w-1/5 py-2 flex flex-col justify-center">
-    <Status bridgeTx={item} />
+    <Status bridgeTx={item} isDesktopOrLarger={isDesktopOrLarger} isMobileDetailsDialog={false}/>
```

Comment:
if we are already passing `isDesktopOrLarger `(should not be passed as prop imho) why do we pass the `isMobileDetailsDialog `as well?

If `isDesktopOrLarger ` is `true` `isMobileDetailsDialog ` will always be `false`. 

I think this needs to done differently

Reply:
Updated the naming from ```isMobileDetailsDialog``` to ```isFromMobileDetailsDialog```

Reply:
Trying a different approach.